assets/js/theme/product/reviews.js

```diff
@@ -24,6 +24,7 @@ export default class {
         const $content = $('#productReviews-content', this.$reviewsContent);
 
         $('.productView-reviewLink').on('click', () => {
+            $('.productView-reviewTabLink').click();
```

Comment:
Should this be `$('.productView-reviewTabLink').trigger('click')`?

Only ask because I recall the change a while back from .click().

<img width="553" alt="Screen Shot 2019-05-14 at 7 01 36 PM" src="https://user-images.githubusercontent.com/5056945/57743702-fb8e3780-767a-11e9-938b-806b6e0c8939.png">

<img width="273" alt="Screen Shot 2019-05-14 at 7 03 51 PM" src="https://user-images.githubusercontent.com/5056945/57743730-16f94280-767b-11e9-8fd8-bd41ffd2b63d.png">

Reply:
Ok, about to make another pull request using that style instead.  Thank you.